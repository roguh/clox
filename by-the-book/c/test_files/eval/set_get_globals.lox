var a = 12; print(a); a = 13; print(a);

var breakfast = "beignets";
var beverage = "covfefe";
breakfast = breakfast + " with " + beverage;
print(breakfast);
