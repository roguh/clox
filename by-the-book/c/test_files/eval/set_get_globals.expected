12
13
beignets with covfefe
